pkg/discovery/discovery.go

```diff
@@ -397,3 +413,24 @@ func setPodStatusAnnotation(client kubernetes.Interface, namespace string, statu
 	_, err = client.CoreV1().Pods(namespace).Patch(podName, types.MergePatchType, patchBytes)
 	return err
 }
+
+func getItemInTree(i *results.Item, root string) *results.Item {
```

Comment:
This was copied from elsewhere; I considered putting it into `results` and exporting it; maybe even as method on the `Item` type. Let me know if you'd prefer that.

Reply:
:+1: I think having just one copy would be good.